crates/re_log_types/src/component_types/tensor.rs

```diff
@@ -404,6 +407,21 @@ impl TensorTrait for Tensor {
             TensorData::F64(_) => TensorDataType::F64,
         }
     }
+
+    fn size_in_bytes(&self) -> usize {
+        match &self.data {
+            TensorData::U8(buf) | TensorData::JPEG(buf) => buf.0.len(),
+            TensorData::U16(buf) => buf.len(),
+            TensorData::U32(buf) => buf.len(),
+            TensorData::U64(buf) => buf.len(),
+            TensorData::I8(buf) => buf.len(),
+            TensorData::I16(buf) => buf.len(),
+            TensorData::I32(buf) => buf.len(),
+            TensorData::I64(buf) => buf.len(),
+            TensorData::F32(buf) => buf.len(),
+            TensorData::F64(buf) => buf.len(),
```

Comment:
Oh wow, it is quite surprising to me that `Buffer<f64>::len()` is the number of bytes, and not the number of elements, but it seems right: https://docs.rs/arrow2/latest/arrow2/buffer/struct.Buffer.html#method.len

Reply:
I filed an issue for this: https://github.com/jorgecarleitao/arrow2/issues/1430